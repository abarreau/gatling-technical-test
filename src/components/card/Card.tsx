import { FC, ReactNode } from 'react';
import styles from './Card.module.css';

/**
 * A card is a container for text, pictures, etc.
 */
export const Card: FC<{ children?: ReactNode }> = ({ children }) => {
    return (<div className={styles.card}>
        { children }
    </div>);
};
