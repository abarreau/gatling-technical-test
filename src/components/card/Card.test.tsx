import renderer from 'react-test-renderer';
import { Card } from './Card';

describe('Card', () => {
    it('should display component', () => {
        const component = renderer.create(
            <Card>My card</Card>
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
