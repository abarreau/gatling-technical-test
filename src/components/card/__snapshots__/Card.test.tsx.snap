// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should display component 1`] = `
<div
  className="card"
>
  My card
</div>
`;
